src/org/usfirst/frc/team694/robot/commands/auton/MiddleSwitchAutonCommand.java

```diff
@@ -0,0 +1,34 @@
+package org.usfirst.frc.team694.robot.commands.auton;
+
+import org.usfirst.frc.team694.robot.commands.DrivetrainRotateDegreesPIDCommand;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ *
+ */
+public class MiddleSwitchAutonCommand extends CommandGroup {
+    public double speed = 0.5;
+    public MiddleSwitchAutonCommand() {
+        addSequential(new DrivetrainMoveInchesEncoderCommand(FieldMap. , speed));
```

Comment:
This won't compile.

You may leave the Switch auton out of this, we may not use it. You can stash your changes and add them in later (maybe make a separate temporary branch to hold your WIP switch auton?)